### ISSUE TITLE
Warp icon in macOS dock does not display illuminated dot below running app

### ISSUE THREAD

**Issue by grempe** [warpdotdev/Warp#1826]
### Discord username (optional)

Glenn#9362

### Describe the bug

Every macOS app I run, without exception to my knowledge, displays a small illuminated dot below the dock icon when the app is running.

Warp does not do this. Without this it does not feel like a native mac application.

### To Reproduce

Open Warp. Observe missing 'running' indicator below app icon.

### Expected behaviour

I should see the illiuminated dot when the app is running in the dock.

### Screenshots

![Screen Shot 2022-09-10 at 16 48 07 PM](https://user-images.githubusercontent.com/117/189501236-580aca7b-4ac7-4915-a551-0efad37c664f.png)

### Operating System

MacOS

### OS Version

12.5.1

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin21.0)

### Warp Version

v0.2022.09.07.14.56.stable_00

### Does this block you from using Warp daily?

No

**Comment by dannyneira** [comment]
Hi @grempe , I'm Danny, Support Engineer at Warp. Sorry to hear you're dealing with this issue. I'm not able to reproduce this on my end in either light or dark mode I see the dot under Warp and I'm using the same version of MacOS/zsh/Warp.

By any chance did you install using `brew install --cask warp` command or .[dmg download](https://app.warp.dev/get_warp) method?
Have you tried uninstalling/reinstalling the Warp, with either method?
If you reinstall and see the same issue, do you mind sending us your warp.log from the Console.app to feedback@warp.dev along with your GH#1826? This will help us investigate the issue.

**Comment by grempe** [comment]
Hi.

tl;dr this issue has now mysteriously resolved itself since I reported it. I now see the dot.

(But, as you can see in my screenshot, when I had multiple Warp windows open, there was no dot. I swear I didn't make it up. :-)

To answer your question, I installed Warp from .dmg, not a brew cask.

Since neither I, nor you, can re-create this now, please feel free to close this issue.

Cheers.

**Comment by dannyneira** [comment]
Thanks for the update @grempe , I'll close this issue but feel free to ping us here again if it returns.
